app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+from app.errors import (NotVaccinatedError,
```

Comment:
start your imports from new line, it increase readability 
```suggestion
from app.errors import (
NotVaccinatedError,
```